### ISSUE TITLE
Body Parser deprecated past on versions of Express > 4.16

### ISSUE THREAD

**Issue by xScoopy** [MakeSchool-Tutorials/Node-Reddit-Clone#39]
Installation and instantiation of 'body parser' shows deprecated status within the IDE. information gathered from 
[stackOverflow](https://stackoverflow.com/questions/24330014/bodyparser-is-deprecated-express-4#:~:text=Explanation%3A%20The%20default%20value%20of,%2Dof%2Dthe%2Dbox.)
shows that instead of pulling in body parser, we can simply use the following:
`app.use(express.urlencoded({ extended: true }));`
`app.use(express.json());`

**Comment by starlightromero** [comment]
Good catch @xScoopy! I just implemented this into the update. It will not be pushed to GH until the end of the term in a few weeks but do know that I am working on updates.
